docs/Installing.asciidoc

```diff
@@ -797,8 +796,8 @@ directories prior registering with openQA instance:
 4. `/var/lib/openqa`
 5. fail if none of above is available
 
-Once worker registers to openQA instance it checks for available job and starts
-accepting websockets commands. Worker accepts jobs as they will come in, there
+Once a worker registers to an openQA instance it checks for available jobs and starts
+accepting websockets commands. Workers accept jobs as they will come in, there
 is no priority, or other ordering, support at the moment.
```

Comment:
This whole section is actually completely wrong. A worker does never check for available jobs. It is possible to prioritize jobs.

So this should be: "Once a worker registers to an openQA instance, a scheduled job (of matching worker class) will be assigned to it. Dependencies between jobs will be considered for ordering the job assignment. Generally, jobs with higher priority (lower priority value) will be assigned first." - Although I'm wondering whether we need to go into that much detail here anyways.

Reply:
I've simplified it to:
"Once a worker registers to an openQA instance, scheduled jobs (of matching
worker class) can be assigned to it. Dependencies between jobs will be
considered for ordering the job assignment."